jni/include/knn_extension/faiss/utils/BitSet.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <faiss/MetricType.h>
+#include <faiss/impl/platform_macros.h>
+#include <limits>
+
+using idx_t = faiss::idx_t;
```

Comment:
using this means that bitset is not dependent and can only be used with Faiss. Should we make this agnostic of any underline library?

Reply:
Let me change it.